src/pages/Content/index.js

```diff
@@ -26,3 +32,28 @@ function click(selector) {
   const element = document.querySelector(selector) || null;
   if (element) element.click();
 }
+
+// a function that listens to click events and stores them in the recordedEvents array
+function attachGlobalEventListeners() {
+
+  // capture all click events
+  document.body.addEventListener('click', function (e) {
+
+    // find the best selector for click target (id/class/tag/attr)
+    const selector = finder(e.target);
```

Comment:
Did you check with event [bubbling/capturing](https://stackoverflow.com/questions/4616694/what-is-event-bubbling-and-capturing)?

Reply:
I can add `{capture: true}` and push the changes, I'm just not sure if it's necessarily better than bubbling.
Let me know

Reply:
No, just check that the click works well with nested components :+1:

Reply:
@MariyaShavshy let me know when you're done.

Reply:
@agamm It works great, I've already checked it a while back in the `console.debug()` section...